driver/src/test/java/org/kaazing/k3po/driver/internal/tls/TlsIT.java

```diff
@@ -44,6 +45,16 @@ public void shouldEstablishConnection() throws Exception {
         k3po.finish();
     }
 
+    @Ignore("Needs JDK 9 to run as it uses SSLParameters#setApplicationProtocols()")
```

Comment:
Can we unignore this by building `k3po` with `JDK 9` but targeting `1.8` for `source` and `target` so it runs on `Java 8` too?